tests/test_selector.py

```diff
@@ -0,0 +1,28 @@
+import molecularnodes as mn
+
+
+def test_select_ca():
+    mol = mn.Molecule.fetch("8H1B")
+    mol.selector.atom_name("CA").chain_id(["A"]).store_named_attribute(
+        "custom_selection"
+    )
+    assert sum(mol.named_attribute("custom_selection")) == 190
+    mol.selector.reset().atom_name("CA").store_named_attribute("custom_selection")
+    assert sum(mol.named_attribute("custom_selection")) == 384
+    (
+        mol.selector.reset()
```

Comment:
I didn't see how you were thinking but this is quite clean as an API. 

This shows the selections work.  I am still not sure how you are imagining tying this together for VIZ. But that is out of scope.

Reply:
Thanks. It'll hopefully become more clear when I can merge others as well and it starts coming together.